extensions/amp-bind/0.1/bind-impl.js

```diff
@@ -172,7 +172,12 @@ export class Bind {
           break;
         }
         const bindings = [];
-        for (const attr of el.attributes) {
+        const attrs = el.attributes;
+        for (let j = 0;; j++) {
```

Comment:
Cache the length:

```js
for (let j = 0, l = attributes.length; j < l; j++) {
}
```

Doing an out-of-bounds access is likely slower than anything else in here.